sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -49,15 +49,19 @@ namespace Azure { namespace Core { namespace Http {
   struct CurlTransportOptions final
   {
     /**
-     * @brief The string for the proxy is passed directly to the libcurl handle without any parsing
+     * @brief The string for the proxy is passed directly to the libcurl handle without any parsing.
+     *
+     * @details libcurl will use system's environment proxy configuration (if it is set) when the \p
```

Comment:
`\p` - does that force a paragraph break? If so, why insert a paragraph break here?

Reply:
no, \p makes the next word to be highlighted (p is for param).  I just want to make emphasis on `Proxy`